Assets/VRTK/Examples/ExampleResources/Prefabs/SDKSetups.prefab

```diff
@@ -5144,12 +5173,12 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   autoPopulateObjectReferences: 1
-  actualBoundaries: {fileID: 1000012704122952}
```

Comment:
_All_ changes beginning here in _this_ file (_including_ this line) should be reverted.

Reason is that these got reset because you don't have all the vendor SDKs installed that VRTK supports. But since this is the _prefab_ I suggest to leave these auto populated references in so people don't freak out because the prefab gets modified all the time. It's better to let Unity only change the prefab _instances_ in a scene, than to change the prefab.